ytrello.rb

```diff
@@ -1,15 +1,3 @@
-# install missing gems using bundler
-bundler_dir = File.expand_path("../.vendor", __FILE__)
-
-if !File.exist?(bundler_dir)
-  puts "Installing needed Rubygems to #{bundler_dir} ..."
-  require "shellwords"
-  if !system "bundle install --path #{Shellwords.escape(bundler_dir)}"
-    $stderr.puts "Installing the needed Ruby gems failed"
-    exit 1
-  end
-end
```

Comment:
Is this necessary for the dockerization? Because removing this would mean that the previous deployment method gets broken.

Reply:
The main problem is that it assumes the bundler_dir for you and force the install if not present while it should just a 
```
require 'rubygems'
require 'bundler/setup'
```
letting to the user the decision about his bundle path, config and so on.

http://bundler.io/v1.12/bundler_setup.html#bundlersetup

We could modify the Dockerfile to explicitly use the .vendor dir avoiding the remove of this chunk of code but not sure if it is the recommended way of bundle use. I was surprised why always that I launched the commands after a build it was installing again all the gems... took me sometime to realize that it was because the code force you to install the gems in a specific path.

Reply:
BTW, I will just remove the global path and it will work fine again not having to remove anything from previous code by now.